onnxruntime/core/providers/rocm/rocm_execution_provider_info.cc

```diff
@@ -22,6 +22,7 @@ constexpr const char* kGpuExternalFree = "gpu_external_free";
 constexpr const char* kGpuExternalEmptyCache = "gpu_external_empty_cache";
 constexpr const char* kMiopenConvUseMaxWorkspace = "miopen_conv_use_max_workspace";
 constexpr const char* kTunableOpEnabled = "tunable_op_enabled";
+constexpr const char* kTunableOpTuning = "tunable_op_tuning";
```

Comment:
Naming: Consider renaming to `enable_tunable_op` and `enable_op_tuning` to be more clear.

Reply:
See https://github.com/microsoft/onnxruntime/pull/15156#issuecomment-1493719115